Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM centos:centos7
+
+EXPOSE 8080
+
+ADD sendEmail-epel-7.repo /etc/yum.repos.d/
+
+RUN yum install -y epel-release && \
+    yum -y --setopt=tsflags=nodocs update && \
+    yum -y --setopt=tsflags=nodocs install httpd \
+                                           nagios \
+                                           telnet \
+                                           supervisor \
+                                           python-jinja2 \
+                                           nagios-plugins-all \
+                                           sendEmail \
+                                           perl-Net-SSLeay \
+                                           perl-IO-Socket-SSL && \
+    yum clean all && \
+    mkdir -p /opt/rhmap/ && \
+    sed -i -e 's/Listen 80/Listen 8080/' /etc/httpd/conf/httpd.conf && \
+    sed -i -e 's|DocumentRoot "/var/www/html"|DocumentRoot "/usr/share/nagios/html"|' -e 's|<Directory "/var/www">|<Directory "/usr/share/nagios/html">|' /etc/httpd/conf/httpd.conf && \
+    touch /supervisord.log supervisord.pid && \
+    mkdir -p /var/log/nagios/rw/ && \
+    mkdir -p /var/log/nagios/spool && \
+    mkdir -p /var/log/nagios/spool/checkresults && \
+    chmod -R 777 /supervisord.log /supervisord.pid /var/log/nagios \
+                    /etc/httpd /etc/passwd /var/log /etc/nagios /usr/lib64/nagios /var/spool/nagios /run /usr/share/httpd /usr/share/nagios && \
+    sed -i -e 's|cfg_file=/etc/nagios/objects/localhost.cfg||' /etc/nagios/nagios.cfg
+
+ADD supervisord.conf /etc/supervisord.conf
+ADD make-nagios-fhservices-cfg.py /opt/rhmap/make-nagios-fhservices-cfg.py
+ADD make-nagios-commands-cfg.py /opt/rhmap/make-nagios-commands-cfg.py
+ADD fhservices.cfg.j2 /opt/rhmap/fhservices.cfg.j2
+ADD commands.cfg.j2 /opt/rhmap/commands.cfg.j2
```

Comment:
You can combine the 4 `ADD` instructions above in a single instruction. That makes builds faster and produces less layers. Even if we squash the images later, this is still beneficial -- even the squashing process becomes faster.

https://docs.docker.com/engine/reference/builder/#/add

```Dockerfile
ADD make-nagios-fhservices-cfg.py make-nagios-commands-cfg.py fhservices.cfg.j2 commands.cfg.j2 /opt/rhmap/

# Or
ADD *-cfg.py *.cfg.j2 /opt/rhmap/

# Or place the files relative to this Dockerfile in such a way that you can reference all of them in a single word.
ADD rhmap/ /opt/rhmap/

# Or have a "root tree" relative to the Dockerfile and add everything in a single instruction:
# (e.g. https://github.com/sclorg/mongodb-container/blob/master/3.2/Dockerfile#L49)
ADD root/ /
```